### PR TITLE
update flextesa docker image to make it arm64 compatible and future-proof by switching to octez-client

### PR DESCRIPTION
**Pull request by gsebil08**
Docker image `oxheadalpha/flextesa:20221026` is ARM64 compatible. It also include the renaming from `tezos-client` to `octez-client`.
This PR aims to update our dockerfile to make it arm64 compatible and future-proof by replacing `tezos-client` references to `octez-client`.

### CODE REVIEW

File: deku-flextesa/Dockerfile [hunk 1]
```diff
@@ -10,4 +10,4 @@ COPY ./kathmandubox /usr/bin/kathmandubox
 # See contract storage in ../networks/flextesa
 COPY ./deku_flextesa_state.tar /tmp/deku_flextesa_state.tar
 RUN tar -xvf /tmp/deku_flextesa_state.tar -C /tmp
-COPY .tezos-client /root
+COPY .octez-client /root
```
Comment (aguillon) [inline deku-flextesa/Dockerfile:13]:
```suggestion
+COPY octez-client /root/.octez-client
```

Comment (gsebil08) [reply deku-flextesa/Dockerfile:13]: Why?

Comment (aguillon) [reply deku-flextesa/Dockerfile:13]: [You're changing the name of deku-flextesa/.tezos-client to deku-flextesa/octez-client](https://github.com/marigold-dev/deku/pull/927/files#diff-31616db27e4a4e1461a6eed444c29808dd0397874c01a841d8fdc271b424a527) (mind the `.` in front of the name). I think it's a good idea, but the client expects a `.octez-client`.

**Comment by aguillon** [comment]
It seems I can't deploy a new contract anymore, or I'm not sure how:
```
storage=$(ligo compile storage "deku-p/src/core/tezos_interop/consensus.mligo" "$(cat networks/flextesa/consensus_storage.mligo)")
contract=$(ligo compile contract "deku-p/src/core/tezos_interop/consensus.mligo")
octez_client --wait 1 originate contract consensus transferring 0 from alice running "$contract" --init "$storage" --burn-cap 2
```

**Comment by gsebil08** [comment]
> It seems I can't deploy a new contract anymore, or I'm not sure how:
> 
> ```
> storage=$(ligo compile storage "deku-p/src/core/tezos_interop/consensus.mligo" "$(cat networks/flextesa/consensus_storage.mligo)")
> contract=$(ligo compile contract "deku-p/src/core/tezos_interop/consensus.mligo")
> octez_client --wait 1 originate contract consensus transferring 0 from alice running "$contract" --init "$storage" --burn-cap 2
> ```

@aguillon since we have figured out it has no relation with this PR ([fix](https://github.com/marigold-dev/deku/commit/4bfe2a637780e035332a4660764f8b750abf0ef8) is already on `main`), I think we can merge this one (that would help people, like me, running M1)

**Pull request merged**